assets/js/components/SaptuneDetails/SaptuneDetails.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+
+import { find, map } from 'lodash';
+
+import BackButton from '@components/BackButton';
+import ListView from '@components/ListView';
+import PageHeader from '@components/PageHeader';
+
+import SaptuneTuningState from './SaptuneTuningState';
+import SaptuneVersion from './SaptuneVersion';
+
+const renderService = (serviceName, services) => {
+  const currentService = find(services, { name: serviceName });
+
+  if (!currentService.enabled) {
+    return <span>-</span>;
+  }
+
+  return (
+    <span>
+      {currentService.enabled}/{currentService.active}
+    </span>
+  );
+};
+
+const renderNote = (noteID) => (
```

Comment:
@dottorblaster gave me a tip to use a element instead of Link as Link is used for internal routing.